partials/nav.php

```diff
@@ -0,0 +1,75 @@
+<?php
+session_start(); // Start the session
+
+if(isset($_SESSION['loggedin']) && $_SESSION['loggedin'] == true){
+  $loggedin = true;
+} else {
+  $loggedin = false;
+}
+
+echo '
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bootstrap demo</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+  </head>
+  <body>
+  
+  <nav class="navbar navbar-expand-lg" style="background-color: #e3f2fd;">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">ChattingApp</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">';
+
+      if($loggedin){
+        echo '
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="Userhomepage.php">Home</a>
+        </li>
+        ';
+      }
+      
+      if(!$loggedin){
+        echo '
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="Homepage.php">Home</a>
+        </li>
+        ';
+      }
+      
+        
+      if($loggedin){
+        echo '
+        <li class="nav-item">
+          <a class="nav-link" href="logout.php">Logout</a>
+        </li>';
+      }
+      
+      if(!$loggedin){
+        echo '
+        <li class="nav-item">
+          <a class="nav-link" href="Userloginpage.php">Login</a>
+        </li>';
+      }
+        
+echo '
+      </ul>
+
+      <form class="d-flex" role="search">
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+        <button class="btn btn-outline-success" type="submit">Search</button>
+      </form>
+    </div>
+  </div>
+</nav>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
+  </body>
+</html>
+';
```

Comment:
You cannot echo a whole HTML code, Thats not a good practice. Find alternative ways to fix this.